README.md

```diff
@@ -1,160 +1,718 @@
+  searchKey: "search-371auk61r2bwqtdzocdgutmg",
+  engineName: "search-ui-examples",
+  hostIdentifier: "host-2376rb"
+});
+
+export default function App() {
+  return (
+    <SearchProvider
+      config={{
+        apiConnector: connector
+      }}
+    >
+      {() => (
+        <div className="App">
+          <Layout
+            header={<SearchBox />}
+            bodyContent={<Results titleField="title" urlField="nps_link" />}
+          />
+        </div>
+      )}
+    </SearchProvider>
+  );
+}
+```
 
-### Requirements
+A search UI is made up of four key areas, which you can expand and customize:
 
-Node: 8.10
-NPM: 6.4
+1. [Connectors](#1-connectors)
+2. [SearchProvider](#2-searchprovider)
+3. [Components](#3-components)
+4. [Styles and Layout](#4-styles-and-layout)
 
-### Mono-repo explanation
+---
 
-This repository is maintained as a Monorepo using [Lerna](https://lernajs.io/).
+### 1. Connectors
 
-Lerna configuration is contained in `lerna.json`.
+Connectors are modules that tell Search UI how to connect and communicate with a particular API.
 
-`/packages` - Contains publishable search-ui npm packages.
-`/examples` - Contains non-publishable examples of search-ui usage. They are declared
-as "packages" in `lerna.json` so that `npx lerna bootstrap` will automatically wire up the
-examples to local dependencies.
+Search UI currently provides two Connectors:
 
-Because all examples are declared as "private", when running lerna commands other than bootstrap, (like `publish` and
-`test`), the `--no-private` flag should be appended.
+1. **Elastic App Search**: [search-ui-app-search-connector](packages/search-ui-app-search-connector)
+2. **Elastic Site Search**: [search-ui-site-search-connector](packages/search-ui-site-search-connector)
 
-Dependencies are declared in a package.json hierarchy.
+The [example search UI](#search-ui) above uses the Elastic App Search Connector:
 
-- / package.json - Dependencies for repo tooling, like `husky` and `lerna`.
-- /packages package.json - Common dev dependencies for all Search UI npm packages. Any dev Common dev dependencies for al that does not need to be called directly in a package level npm command
-  can be declared here.
-- /packages/{package_name} package.json - Package specific dependencies.
+```js
+const connector = new AppSearchAPIConnector({
+  searchKey: "search-371auk61r2bwqtdzocdgutmg",
+  engineName: "search-ui-examples",
+  hostIdentifier: "host-2376rb"
+});
+```
 
-Note that we do not encourage "hoisting" dependencies through lerna. This WILL
-cause the examples applications to error out from dependency version conflicts.
+Search UI can connect to **any** web based Search API. Read [the wiki](https://github.com/elastic/search-ui/wiki/Build-Your-Own-Connector) for more information.
 
-<a id="install"></a>
+### 2. SearchProvider
 
-### Installing dependencies
+The `SearchProvider` object will tie all of your Components together so that they
```

Comment:
I think there's a couple of things we should call out here.

1. It's where you configure your UI.
2. It is a state manager, and it exposes "State" and "Actions" which you can use within your application. This ties into the concept of "Headless" above.